src/ngrok/index.ts

```diff
@@ -98,6 +99,11 @@ const tunnelsFromConfig = (tunnels: { [key: string]: INgrokOptions }) => {
   });
 };
 
+const getActiveTunnels: (api: any) => Promise<Tunnel[]> = async (api: any) => {
+  const response = ((await api.get("api/tunnels")) as unknown) as string;
```

Comment:
I think it's a bit better to use build in parsing:
```ts
const response = await (api.get('api/tunnels', { json: true }) as RequestPromise<TunnelsResponse>);
```

This requires to install `@types/request-promise-native`, but this is correct `ngrok` is [using](https://github.com/bubenshchykov/ngrok/blob/74d0509ef3f63f9a7b2415ba8f992e1b5a3cde5f/index.js#L1) `request-promise-native` to wrap `request` calls.

This will avoid casting to `unknown` ;)

Reply:
Thanks, I was wondering why I had to do this. Still pretty new to getting TypeScript to do what I want.